apps/web/src/server/applications/case/documentation/applications-documentation.service.js

```diff
@@ -243,3 +248,15 @@ export const publishCaseDocumentationFiles = async (caseId, documents, username)
 		return { errors: { msg: 'Your documents could not be published, please try again' } };
 	}
 };
+
+//document search
+/**
+ * @param {ApplicationsSearchResultsBody} payload
+ * @returns {Promise<PaginatedApplicationsResponse>}
```

Comment:
this is probably wrong, isn't it? your service is returning a paginated list of documents and not applications.